packages/mobile/src/screens/root-screen/RootScreen.tsx

```diff
@@ -16,6 +16,7 @@ import { UpdateRequiredScreen } from 'app/screens/update-required-screen/UpdateR
 import { enterBackground, enterForeground } from 'app/store/lifecycle/actions'
 
 import { AppDrawerScreen } from '../app-drawer-screen'
+import { RestartRequiredScreen } from '../update-required-screen/RestartRequiredScreen'
```

Comment:
Nit but most screen directories export screen from index. Curious your thoughts on this patter though

Reply:
oOoo ok thats cool with me i just got lazy. ill make subdirs for these two components :)